Ejercicio 1/Ejercicio 1/Program.cs

```diff
@@ -10,6 +10,8 @@ class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Helloooo World");
```

Comment:
I suggest you add a ReadKey method to stop the application and can see the message.

Reply:
Done.